### ISSUE TITLE
Install ConfigSpace failed: Command "python setup.py egg_info" failed with error code 1

### ISSUE THREAD

**Issue by mu-triv** [automl/ConfigSpace#132]
I install ConfigSpace v.0.4.11 with requirements.txt of autosklearn, but the installation failed:

```bash
  Downloading https://files.pythonhosted.org/packages/be/07/ebb7c6c6d03c82ffe6314161f2c8da32599c841e74b08f5c23812fee0e6f/ConfigSpace-0.4.11.tar.gz (957kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-bu2wj2w5/ConfigSpace/setup.py", line 95, in <module>
        long_description=read_file("README.md"),
      File "/tmp/pip-build-bu2wj2w5/ConfigSpace/setup.py", line 11, in read_file
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1060: ordinal not in range(128)
```
This issue does not exist in v.0.4.10

**Comment by mfeurer** [comment]
Thanks for reporting this. The README indeed contains a non-ASCII character: ü

As a hotfix you can either change the default encoding of your machine to UTF-8, patch the call in https://github.com/automl/ConfigSpace/blob/master/setup.py#L11, or temporarily remove the ü from the readme file prior to installing the ConfigSpace.

**Comment by JonasMedu** [comment]
thx @mu-triv for referecing this here.
i had this error just now, trying to install auto-sklearn from pip on a docker container and could not figure out the UnicodeDecodeError.
Now i fixed the 
```UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1060: ordinal not in range(128)```
error ```pip3 install auto-sklearn``` through by addind ```ConfigSpace==0.4.10``` to the requirements! :)

**Comment by JonasMedu** [comment]
thx @mu-triv for referecing this here.
i had this error just now, trying to install auto-sklearn from pip on a docker container and could not figure out the `UnicodeDecodeError`.
Now i fixed the
```
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1060: ordinal not in range(128)
```
error 
with a dedicated
```
RUN pip3 install ConfigSpace==0.4.10
RUN pip3 install auto-sklearn
```
and it works, yupiee

**Comment by mfeurer** [comment]
Actually, there has been a duplicate in #137 that has been fixed in #138.
